lib/wicked_pdf/pdf_helper.rb

```diff
@@ -19,27 +19,28 @@ def self.prepended(base)
 
       base.class_eval do
         after_action :clean_temp_files
+      end
+    end
 
-        alias_method :render_without_wicked_pdf, :render
-        alias_method :render_to_string_without_wicked_pdf, :render_to_string
-
-        def render(options = nil, *args, &block)
-          render_with_wicked_pdf(options, *args, &block)
-        end
+    def render(options = nil, *args, &block)
+      render_with_wicked_pdf(options, *args, &block)
+    end
 
-        def render_to_string(options = nil, *args, &block)
-          render_to_string_with_wicked_pdf(options, *args, &block)
-        end
-      end
+    def render_to_string(options = nil, *args, &block)
+      render_to_string_with_wicked_pdf(options, *args, &block)
     end
 
     def render_with_wicked_pdf(options = nil, *args, &block)
       if options.is_a?(Hash) && options.key?(:pdf)
         log_pdf_creation
         options[:basic_auth] = set_basic_auth(options)
         make_and_send_pdf(options.delete(:pdf), (WickedPdf.config || {}).merge(options))
-      else
+      elsif respond_to?(:render_without_wicked_pdf)
```

Comment:
I'm not sure I understand. What determines whether this returns `true` or `false`?